shinysdr/telemetry.py

```diff
@@ -109,21 +109,26 @@ def __new__(cls, *args, **kwargs):
 class ITelemetryObject(Interface):
     """
     An object that can be in an TelemetryStore.
+
+    Examples: an APRS station, aircraft, etc. The state of an ITelemetryObject
+    is updated through receiving ITelemetryMessage.
     """
-    
+
     def receive(message):
         """
-        TODO document
+        Update state according to the received ITelemetryMessage.
         """
-    
+
     def is_interesting():
         """
-        Return whether this object should be shown to the client. The value should change only when a message is receive()d.
+        Return whether this object should be shown to the client. The value
+        should change only when a message is receive()d.
```

Comment:
> Your own written style guidelines say to follow PEP8:

Oops, I forgot to write that in the exceptions. Sorry! Will fix!

> And as an aside, it's no fun as a contributor to work on a patch and then get a bunch of style comments which require a bunch of tedious editing.

Indeed, and that's why I decided to commit most the fixes myself. This one I just forgot about because I was reviewing your two commits separately. Again, sorry.

Yours are the first nontrivial contributions I've received.

> Set up Travis or similar to run it automatically for any PR in case the author neglects to do so.

I'll look into Travis for at least my existing tests and lint.